_episodes/03-loops.md

```diff
@@ -349,75 +349,69 @@ u
 
 We now have almost everything we need to process
 multiple data files with our `analyze` script.
-You'll notice that our data files are named
-`inflammation-01.csv`, `inflammation-02.csv`, etc.
-Let's write a loop that tries to print the names of each one of our files:
 
-~~~
-for idx = 1:12
-    file_name = sprintf('inflammation-%d.csv', idx);
-    disp(file_name);
-end
-~~~
+We need to generate a list of data files to process,
+and then we can use a loop to repeat the analysis for each file.
+
+We can use the `dir` command to return a **structure array** containing
+the names of the files in the `data` directory.
+Each element in this *structure array* is a **structure**, containing
+information about a single file in the form of named **fields**.
+
+```
+files = dir('data/*.csv')
```

Comment:
Maybe `inflammation-*.csv` instead?

Reply:
I've just noticed we moved too quickly with the changes. Subsequent developments of this example still use `data/*.csv` as the match pattern. I'll fix it so they match.